### ISSUE TITLE
What's going on with Chromium size, it has grown three times in the latest minor release (1.17.0 -> 1.18.0)

### ISSUE THREAD

**Issue by limonte** [puppeteer/puppeteer#4610]
I'm trying to install `puppeteer` on Linux and was shocked by the Chromium size:

```sh

npm i puppeteer@1.17.0

Downloading Chromium r662092 - **111.6 Mb**
```

```sh

npm i puppeteer@1.18.0

Downloading Chromium r669486 - **347.1 Mb**
```

Is Chromium growing so rapidly or is it a bug?

**Comment by sudofox** [comment]
Yeah, I was wondering the same thing. Redeploying something using Puppeteer and Chromium is now over one-third of a GB in size

**Comment by aslushnikov** [comment]
Whoa, that's a bug!

**Comment by aslushnikov** [comment]
Filed at https://bugs.chromium.org/p/chromium/issues/detail?id=977716

**Comment by limonte** [comment]
Thank you @aslushnikov!

As I can see the reported issue is already fixed.

---

I have an offtopic question: since you work in Google do you maybe know why isn't Chromium repo on Github? I mean GitHub is the "default" place to collaborate on software. The current UI for Chromium's bug-tracker looks terrible (in my eyes) and I personally can't figure out what's happening with repo and therefore can't contribute.

**Comment by stevenvachon** [comment]
It's been running Travis CI out of memory too. Reverting to `1.17.0` fixes it.

**Comment by econner** [comment]
It also is causing slug sizes on Heroku to be too large if you run puppeteer there. Confirm that reverting to `1.17.0` also fixes it.

**Comment by wavded** [comment]
Same issue, reverting to `1.17.0` fixed it for me too.

**Comment by aslushnikov** [comment]
We released v1.18.1 with the fix for the bug. 

Sorry for this regression!
